### PR TITLE
Args: Add `remains` and fix a typo in the `len` docs

### PR DESCRIPTION
**Pull request by anyputer**
Let me know if `option_rest` isn't a good name.

### CODE REVIEW

File: src/framework/standard/args.rs [hunk 1]
```diff
@@ -746,13 +746,27 @@ impl Args {
         &self.message[start..]
     }
 
+    /// Starting from the offset, return the remainder of available arguments.
+    ///
+    /// Returns `None` if there are no remaining arguments.
+    #[inline]
+    pub fn option_rest(&self) -> Option<&str> {
```
Comment (arqunis) [inline src/framework/standard/args.rs]: I propose naming this a synonym of "rest":
```suggestion
    pub fn remains(&self) -> Option<&str> {
```

File: src/framework/standard/args.rs [hunk 2]
```diff
@@ -746,13 +746,27 @@ impl Args {
         &self.message[start..]
```
Comment (arqunis) [inline src/framework/standard/args.rs]: It's just barely out of reach, but could you reimplement `rest` in terms of `remains`? I.e.:
```rust
pub fn rest(&self) -> &str {
    self.remains().unwrap_or("")
}
```

**Review by arqunis — CHANGES_REQUESTED** [review]

File: src/framework/standard/args.rs (hunk 2 above)
Comment (anyputer) [reply src/framework/standard/args.rs]: I have thought of that but I'm worried about any possible performance implications. Sure thing.

Comment (arqunis) [reply src/framework/standard/args.rs]: Don't worry. If anything, rustc and, especially, llvm are very smart in inlining/optimizing unnecessary calls.
If you look at [this output from godbolt](https://godbolt.org/z/_XsW-r), you can see `rest` and `rest2` are pretty much identical in their assembly.

**Review by arqunis — APPROVED** [review]

**Pull request merged**